release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,389 @@
+{product-title-short} creates SBOMs of the Analyzed type and complies with the Software Package Data Exchange (SPDX) 2.3 specification.
+
+For more information about the SBOMs of the Analyzed type, see
+link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[Types of SBOM Documents] (America's Cyber Defense Agency documentation).
+
+////
+For information about how to use {product-title-short} to generate SBOMs, see xref:../operating/manage-vulnerabilities/scanner-generate-sbom.adoc#scanner-generate-sbom[Generating SBOMs from scanned images].
+////
+
+[NOTE]
+====
+Generating SBOMs from images analyzed through secured clusters by using the delegated scanning feature is currently not supported. It is anticipated that an upcoming release will support this feature.
+====
+
+//ROX-17535
+[id="automatic-certificate-rotation-for-secured-clusters_{context}"]
+=== Automatic certificate rotation for secured clusters
+
+{product-title-short} 4.7 introduces the automatic rotation of certificates.
+{product-title-short} secured cluster components use these certificates to securely communicate with the Central service.
+Your secured cluster continues to function even if the initBundle and its bootstrap credentials are revoked, as these certificates are separate from the bootstrap credentials in the initBundle.
+
+For more information, see xref:../configuration/reissue-internal-certificates.adoc#reissuing-internal-certificates-for-secured-clusters-by-using-automatic-certificate-renewal_reissue-internal-certificates[Reissuing internal certificates for secured clusters by using automatic certificate renewal].
+
+//ROX-14616
+[id="protect-and-run-secured-clusters-on-red-openshift-on-ibm-cloud_{context}"]
+=== Protect and run secured clusters on Red{nbsp}OpenShift on IBM Cloud
+
+In {product-title-short} 4.7, you can now protect {product-roibm} and run secured clusters on {product-roibm}.
+
+//ROX-13542
+[id="use-the-short-lived-oidc-credentials-from-microsoft-azure_{context}"]
+=== Use the short-lived OIDC credentials from Microsoft Azure
+
+With {product-title-short} 4.7, you can now use short-lived OpenID Connect (OIDC) credentials. The Microsoft Azure Identity Provider provides these credentials, and you can use these credentials to access the services in Microsoft Azure, such as the Azure Container Registry and Azure Sentinel.
+
+This feature builds on the {product-title-short} 4.4 release, which introduced the ability to use short-lived OIDC credentials from Amazon Web Services (AWS) and Google Cloud Platform (GCP) cloud providers to authenticate to the services hosted on these platforms.
+
+For more information, see xref:../integration/integrate-with-image-registries.adoc#manual-configuration-image-registry-acr_integrate-with-image-registries[Manually configuring Microsoft Azure Container Registry].
+
+[id="certifications_{context}"]
```

Comment:
I think these were mentioned in 4.6 so not sure why we are mentioning again? Maybe accidentally left in while cutting and pasting? https://docs.openshift.com/acs/4.6/release_notes/46-release-notes.html#certifications_release-notes-46